src/FeeComponents.js

```diff
@@ -0,0 +1,135 @@
+import * as proto from "@hashgraph/proto";
+export default class FeeComponents {
+    /**
+     * @param {object} props
+     * @param {Long} props.min
```

Comment:
Unfortunately all these types are nullable which means you'd need to surround the name in square brackets: `[props.min]` Also, _iirc_ `props` should be surrounded in square brackets as well `[props]`. Then in the constructor you'd give it a default value of `{}` `constructor(props = {})` This will require you to go through each field and only set it if non-null.

Reply:
This applies to just FeeComponents right?